test_suite/cloud/test_aws.py

```diff
@@ -301,6 +301,10 @@ def test_required_packages_are_installed(self, host):
     @pytest.mark.pub
     @pytest.mark.run_on(['rhel'])
     def test_rhui_pkg_is_installed(self, host):
+        with host.sudo():
+            # Workaround for "yum and rpm get a db5 error"
+            assert host.run('rm -f /var/lib/rpm/__db.*').succeeded, 'failed to remove the RPM db files'
+            assert host.run('rpm --rebuilddb').succeeded, 'failed to rebuild the RPM db files'
```

Comment:
I'm, thinking if we really want to stop test case execution here and make it fail... maybe we can just print a message about what went wrong at this stage but continue trying to check for the RPMs?

Reply:
I would keep the assert here for some reasons:
1. If this is actually the case that the DB is broken and this is why this test fails - then it is not really necessary to continue cause it will fail anyway.
2. If we find out that the assert works fine but still have issues with retrieving data from the DB we'll probably need to change the solution for it

Reply:
Thanks @sshmulev , makes total sense!